### PR TITLE
DOC: Bump fastparquet version

### PR DESCRIPTION
**Pull request by TomAugspurger**
Should we just bump the min version to 0.2.1? It's quite new, but I think reading datetime data is somewhat common :) It'll save us bug reports.

### CODE REVIEW

**Comment by TomAugspurger** [comment]
cc @jorisvandenbossche this fixes the doc error you were seeing.

**Review by jorisvandenbossche — APPROVED** [review]
Looks good, let's wait on the doc build to check if it actually worked

**Review by jreback — CHANGES_REQUESTED** [review]
is this *only* for docs? may have to update the parquet doc section as well.

**Comment by jorisvandenbossche** [comment]
> is this only for docs? may have to update the parquet doc section as well.

For now, this PR is only about the doc build. But if the suggestion of Tom is followed to bump the fastparquet version, we should also update the docs (although, if we decide to not nump the minimum version, we should maybe still mention that for datetime data, the latest version is required)

**Comment by TomAugspurger** [comment]
I've made the changes to bump the min version to 0.2.1 locally, but I think there's an issue with the conda-forge package. Going to fix that before pushing again.

**Comment by jreback** [comment]
why are you bumping for parquet reading? (we had already bumped for this release I think).

**Comment by jreback** [comment]
its fine. we don't usually bump up this fast, but ok.

**Comment by jreback** [comment]
but would need to change the tests & docs

**Comment by TomAugspurger** [comment]
https://github.com/pandas-dev/pandas/pull/24590#issuecomment-451216152

but fixing the CF package first.

**Comment by jreback** [comment]
> but fixing the CF package first.

what is this?

**Comment by TomAugspurger** [comment]
CF=conda-forge. There was an issue with the 0.2.1 build:
https://github.com/conda-forge/fastparquet-feedstock/pull/21

**Comment by jreback** [comment]
ahh ok

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
ok merge ok then

**Comment by jreback** [comment]
merge?

**Comment by jorisvandenbossche** [comment]
No , the problem is not yet fixed (see the doc build log on travis)

Possibly due the conda-forge package problem? (@TomAugspurger the conda list output indicates it installed fastparquet 0.2.1 from conda-forge, but then the error is still there and the `pd.sho_versions()` gives fastparquet 0.1.6 ...)

**Comment by jreback** [comment]
ok cool.

**Comment by TomAugspurger** [comment]
The fastparquet 0.2.1 conda-forge package was accidentally packaging 0.1.6 (but labeling it as 0.2.1), so the error persisted.

The linux builds for a corrected fastparquet are just finishing now. I'll clean up my version bump and push.

File: ci/deps/azure-macos-35.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ dependencies:
   - numpy=1.12.0
   - openpyxl=2.5.5
   - pyarrow
-  - fastparquet
```
Comment (jreback) [inline ci/deps/azure-macos-35.yaml]: can you update environment.yaml as well

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by jorisvandenbossche** [comment]
Some travis builds are failing (didn't check why), but just checked that the doc build is now working (the error is gone)

**Comment by TomAugspurger** [comment]
The travis build is failing because not all the conda packages are done building. Some are still pulling in the (bad) 0.2.1 fastparquet.

I'll let those finish up before pushing again (with the dev env update).

**Comment by TomAugspurger** [comment]
FYI, conda-forge stopped building packages for Python 3.5 on MacOS, so I've removed fastparquet from that env.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks!

**Pull request merged**